app/controllers/chargeback_assignment_controller.rb

```diff
@@ -13,7 +13,7 @@ def self.table_name
 
   def index
     @title = _("Chargeback Assignments")
-    @tabform = "Compute"
+    @tabform = params[:type] && ChargebackRate::VALID_CB_RATE_TYPES.include?(params[:type]) ? params[:type] : "Compute"
```

Comment:
Is this for tab changings? I'd rather go with `params[:tab]` then. cc @himdel

Reply:
What do you think @himdel, I kinda remember that we are already using `tab` elsewhere.

Reply:
@skateman we have `tab`, `tab_id` in few places where we use html tabs in User settings, Reports editor and some other screens.

Reply:
@himdel @skateman do you want me to change `params[:type]` to `params[:tab]` ?

Reply:
@skateman @himdel please review, changed `params[:type]` to `params[:tab]`